scalameta/quasiquotes/shared/src/main/scala-3/scala/meta/internal/quasiquotes/ReificationMacros.scala

```diff
@@ -167,29 +167,23 @@ class ReificationMacros(using val topLevelQuotes: Quotes) {
   private def throwSourceFileError() = // should never be reached
     throw new Exception("Source file contents could not be read while expanding quasiquote")
 
-  private def metaInput() = {
-    val pos = Position.ofMacroExpansion
+  private def metaInput(strCtx: Expr[StringContext]) = {
+    val pos = strCtx.asTerm.pos
     val reflectInput = pos.sourceFile
     val content = new String(reflectInput.content.getOrElse(throwSourceFileError()))
-    val start = {
```

Comment:
why is start replaced with pos.start, but end is not replaced with pos.end? judging by code in isMultiline, pos.start points to quotes, yet we used to skip them in `start` (and still do in `end`).

Reply:
@jchyb not sure if you saw this question